wasm/leb128/read.go

```diff
@@ -0,0 +1,73 @@
+package leb128
```

Comment:
please add a license header.

please also add a brief documentation blurb about the `leb128` package.